common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -15,18 +15,22 @@
  */
 package io.netty.util.internal;
 
+import io.netty.util.SuppressForbidden;
 import io.netty.util.internal.logging.InternalLogger;
 import io.netty.util.internal.logging.InternalLoggerFactory;
 
-import java.io.Closeable;
 import java.io.ByteArrayOutputStream;
+import java.io.Closeable;
 import java.io.File;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
+import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 import java.net.URL;
+import java.nio.file.FileStore;
```

Comment:
@johnou good point!

@mkw we need to use reflection here or move it to another class that is only loaded if java version >= 7